CMakeLists.txt

```diff
@@ -238,8 +238,8 @@ if (NGRAPH_UNIT_TEST_ENABLE)
     add_subdirectory(test)
     message(STATUS "unit tests enabled")
 else()
-    message(STATUS "unit tests disabled")
+    add_subdirectory(test/util)
+    message(STATUS "unit tests disabled, but libngraph_test_util.so will be built")
```

Comment:
Leave the original message. The new part of the message falls under the "too much information" (TMI) category.